src/azure-cli/azure/cli/command_modules/acr/token.py

```diff
@@ -0,0 +1,205 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.util import CLIError
+from ._utils import get_resource_group_name_by_registry_name
+
+
+def acr_token_create(cmd,
+                     client,
+                     registry_name,
+                     token_name,
+                     scope_map_name,
+                     status=None,
+                     resource_group_name=None):
+
+    resource_group_name = get_resource_group_name_by_registry_name(cmd.cli_ctx, registry_name, resource_group_name)
+
+    from ._utils import get_resource_id_by_registry_name
+
+    token_create_parameters = {
```

Comment:
Avoid hardcoded scheman. Please use `get_models('Token')` to construct the object using the schema.